google/cloud/pubsub/subscription_admin_client.h

```diff
@@ -31,6 +31,9 @@ GOOGLE_CLOUD_CPP_INLINE_NAMESPACE_BEGIN
 /**
  * Performs subscription administrative operations in Cloud Pub/Sub.
  *
+ * @deprecated Please use #google::cloud::pubsub_admin::SubscriptionAdminClient
+ *     and #google::cloud::pubsub_admin::SubscriptionAdminClient instead.
```

Comment:
The Doxygen documentation recommends `\ref` (aka `@ref`) to link classes:

https://www.doxygen.nl/manual/autolink.html

Can you run the `publish-docs-pr` build locally and see if the generated `HTML` files look right to you?

Reply:
Ack, will run locally

Reply:
This is what it looks like  (which I think is fine)

```
<div class="line"><a id="l00031" name="l00031"></a><span class="lineno">   31</span><span class="comment">/**</span></div>
<div class="line"><span class="lineno">   32</span><span class="comment"> * Performs subscription administrative operations in Cloud Pub/Sub.</span></div>
<div class="line"><span class="lineno">   33</span><span class="comment"> *</span></div>
<div class="line"><span class="lineno">   34</span><span class="comment"> * @deprecated Please use \ref</span></div>
<div class="line"><span class="lineno">   35</span><span class="comment"> * google::cloud::pubsub_admin::SubscriptionAdminClient and \ref</span></div>
<div class="line"><span class="lineno">   36</span><span class="comment"> * google::cloud::pubsub_admin::SubscriptionAdminClient instead.</span></div>
<div class="line"><span class="lineno">   37</span><span class="comment"> *</span></div>
<div class="line"><span class="lineno">   38</span><span class="comment"> * Applications use this class to perform subscription administrative operations</span></div>
<div class="line"><span class="lineno">   39</span><span class="comment"> * on [Cloud Pub/Sub][pubsub-doc-link].</span></div>
```

Reply:
I feel like they should be hyperlinked with `<a>` tags. What does the html look like rendered in the browser?